hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -78,7 +79,12 @@ private void handleMap(HttpGetCommand command, String uri) {
         int indexEnd = uri.indexOf('/', URI_MAPS.length());
         String mapName = uri.substring(URI_MAPS.length(), indexEnd);
         String key = uri.substring(indexEnd + 1);
-        Object value = textCommandService.get(mapName, key);
+        Object value;
+        if(MAP_ENTRIES_COMMAND.equalsIgnoreCase(key)){
```

Comment:
Hi @onderaycicek 
New addition should not be a problem. 
We had a quick discussion internally. console does not sound straightforward to us, although it  is ok for your use case.

Can you make it as follows ?
```
/hazelcast/rest/maps-entries/mapName/entries
```
I think, It can be implemented easily without breaking old behaviour. I downside is this change will not be sync with queue.

In queue we can get the size as
```
/hazelcast/rest/maps/queueName/size
```
Note: To `poll` from queue, one needs to give integer value as timeout value . this is why `size` was not a problem in `queue`, but a problem in `map`.

I think we should add the following also for queue, just for the sake symmetry.
```
/hazelcast/rest/queues-size/queueName/command/size
```

Later any command can be added following same pattern.